zebra-consensus/src/transaction.rs

```diff
@@ -481,6 +483,13 @@ where
                 let utxo = if let Some(output) = known_utxos.get(outpoint) {
```

Comment:
```suggestion
                // Currently, Zebra only supports known UTXOs in block transactions.
                // But it might support them in the mempool in future.
                let utxo = if let Some(output) = known_utxos.get(outpoint) {
```